node/pkg/reporter/app_test.go

```diff
@@ -11,110 +11,75 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+func TestFetchConfigs(t *testing.T) {
+	configs, err := fetchConfigs()
+	if err != nil || configs == nil || len(configs) == 0 {
+		t.Fatalf("error getting configs: %v", err)
+	}
```

Comment:
it's not a big deal, just an optional comment
you can probably use assert.Nil here

something like
```
assert.Nil(t, err)
```

Reply:
if you don't insist, i think i'll keep it like this for now since it's a nice one liner :)

Reply:
sure

Reply:
but in this case we can't tell between by which of 3 cases it failed